.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  root: true,
+  extends: ['plugin:@typescript-eslint/recommended', 'next/core-web-vitals', 'prettier'],
+  plugins: ['unused-imports'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: './tsconfig.json'
+  },
+  rules: {
+    '@typescript-eslint/no-unused-vars': 'error',
+    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-unsafe-call': 'error',
+    '@typescript-eslint/no-unsafe-member-access': 'error',
+    '@typescript-eslint/no-unsafe-return': 'error',
+    'unused-imports/no-unused-imports': 'error',
+    'unused-imports/no-unused-vars': 'off',
```

Comment:
It is important because if there is no L16, the error about 'use client' occurs.
It may be in my case.
https://qiita.com/yamatai12/items/80fcea6c53991f9d7b3f#%E5%95%8F%E9%A1%8C%E7%82%B9